docs/processes/release/tasks/docs.md

```diff
@@ -374,21 +374,19 @@ Note: This is also referenced below [Upload API documentations](docs.md#upload-a
 Build the API documentation pages for the new release:
 
 ```
-ant releaseDocs
+release_tools/github_release.sh
```

Comment:
1. `release_tools` -> `releases_tools`
2. Make explicit the prior assumption that we're in the icu4j directory

```suggestion
cd icu4j
releases_tools/github_release.sh
```